.github/workflows/branchOffCadenzaVersion.yml

```diff
@@ -0,0 +1,66 @@
+name: Branch off Cadenza Version
+
+on:
+  workflow_dispatch:
+    inputs:
+      cadenza-version:
+        type: string
+        required: true
+        description: |
+          Next Cadenza Main Version
+          (e.g. '10.2' for setting `main` branch version to 10.2.0-dev as corresponding major-minor version of in Cadenza 10.2.)
```

Comment:
"of in" does not look right. Proposal:
> e.g. '10.2' for setting the main branch up for development against Cadenza 10.2. This would make 10.2.0-dev the new main branch version.

Reply:
Thanks, will adopt it!